elogia_sale/models/elogia_sale.py

```diff
@@ -631,6 +670,9 @@ def calc_kpi(self):
                     record.margin_ml = (record.billed_revenue - sum_not_client) / record.currency_id.rate
             record.purchase_ml = - (sum_not_client / record.currency_id.rate) if record.currency_id else 0
             record.margin_factor_ml = record.margin_ml - record.fee_revenue_ml
+            record.objective_month = obj_objective_id.amount_period if obj_objective_id else 0
```

Comment:
mejor así:
`record.objective_month = obj_objective_id and obj_objective_id.amount_period or 0`